content/docs/platform/integrations/email/writing-email-template.mdx

```diff
@@ -77,3 +107,44 @@ Show block can be used to conditionally show content based on a condition. Use t
 />
 
 In above example, if subscriber has `showTracking` field set to `true` in custom `data` attribute, then the `Track your order` button will be shown in the email sent to the subscriber. Similarly, payload variables can be used to conditionally show the content.
+
+## Code editor (custom HTML)
+
+Novu’s custom HTML for email lets you build fully custom email templates by writing raw HTML combined with full LiquidJS support for dynamic data binding, control flow, and personalization. This gives you full control over layout, structure, and rendering, beyond the capabilities of Novu's block editor.
```

Comment:
`Novu’s custom HTML for email lets` this beginning sound weird
```suggestion
Custom HTML Editor lets you build email templates by writing raw HTML combined with full LiquidJS support for dynamic data binding, control flow, and personalization. This gives you full control over layout, structure, and rendering, beyond the capabilities of Novu's block editor.
```
or smth similar